cmd/tikv-ctl/src/main.rs

```diff
@@ -1117,913 +1118,700 @@ fn warning_prompt(message: &str) -> bool {
+        skip_tombstone: bool,
+    },
+}
+
+#[derive(StructOpt)]
+enum FailCmd {
+    /// Inject failures
+    Inject {
+        /// Inject fail point and actions pairs. E.g. tikv-ctl fail inject a=off b=panic
+        args: Vec<String>,
+
+        #[structopt(short = "f")]
+        /// Read a file of fail points and actions to inject
+        file: Option<String>,
+    },
+    /// Recover failures
+    Recover {
+        /// Recover fail points. Eg. tikv-ctl fail recover a b
+        args: Vec<String>,
+
+        #[structopt(short = "f")]
+        /// Recover from a file of fail points
+        file: Option<String>,
+    },
+    /// List all fail points
+    List {},
+}
+
+#[derive(StructOpt)]
+enum EncryptionMetaCmd {
+    /// Dump data keys
+    DumpKey {
+        #[structopt(long, use_delimiter = true)]
+        /// List of data key ids. Dump all keys if not provided.
+        ids: Option<Vec<u64>>,
+    },
+    /// Dump file encryption info
+    DumpFile {
+        #[structopt(long)]
+        /// Path to the file. Dump for all files if not provided.
+        path: Option<String>,
+    },
+}
+
+#[derive(StructOpt)]
+enum UnsafeRecoverCmd {
+    /// Remove the failed machines from the peer list for the regions
+    RemoveFailStores {
+        #[structopt(
+            short = "s",
+            use_delimiter = true,
+            require_delimiter = true,
+            value_delimiter = ","
+        )]
+        /// Stores to be removed
+        stores: Vec<u64>,
+
+        #[structopt(
+            required_unless = "all-regions",
+            conflicts_with = "all-regions",
+            short = "r",
+            use_delimiter = true,
+            require_delimiter = true,
+            value_delimiter = ","
+        )]
+        /// Only for these regions
+        regions: Option<Vec<u64>>,
+
+        #[structopt(long)]
+        /// Promote learner to voter
+        promote_learner: bool,
+
+        // `regions` must be None when `all_regions` is present,
+        // so we left `all_regions` unused.
+        #[allow(dead_code)]
+        #[structopt(required_unless = "regions", conflicts_with = "regions", long)]
+        /// Do the command for all regions
+        all_regions: bool,
+    },
+    /// Remove unapplied raftlogs on the regions
+    DropUnappliedRaftlog {
+        #[structopt(
+            required_unless = "all-regions",
+            conflicts_with = "all-regions",
+            short = "r",
+            use_delimiter = true,
+            require_delimiter = true,
+            value_delimiter = ","
+        )]
+        /// Only for these regions
+        regions: Option<Vec<u64>>,
+
+        // `regions` must be None when `all_regions` is present,
+        // so we left `all_regions` unused.
+        #[allow(dead_code)]
+        #[structopt(required_unless = "regions", conflicts_with = "regions", long)]
+        /// Do the command for all regions
+        all_regions: bool,
+    },
+}
+
+fn main() {
+    let opt = Opt::from_args();
```

Comment:
Does it look like this line does not return the value we want?

ref https://github.com/tikv/tikv/issues/11514